File: Jeong-Yunmi/1/8595.py
#8595 히든넘버

num, word = input(), input()
num_list = []
temp = ""

for i in word:
    if i.isdigit():
        temp += i
    else:
        if temp != "":
            num_list.append(int(temp))
            temp = ""
if word[-1].isdigit():
    num_list.append(int(temp))

print(sum(num_list))

File: d0ubleho/1/8595.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	long long length, sum = 0;
	string word;
	string list = "", temp = "";

	cin >> length >> word;

	for (int i = 0; i < length; i++) {
		if (isdigit(word[i])) {
			list.append(1, word[i]);
		}
		else {
			list.append(1, ' ');
		}
	}

	for (char ch : list) {
		if (isdigit(ch)) {
			temp += ch;
		}
		else if (temp.length() > 0) {
			sum += stoi(temp);
			temp = "";
		}
	}

	if (temp.length() > 0) {
		sum += stoi(temp);
	}

	cout << sum;

	return 0;
}